.env.example

```diff
@@ -54,6 +54,7 @@ REDIS_PORT=6379
 DJANGO_SETTINGS_MODULE=redbox_app.settings
 DEBUG=True
 DJANGO_SECRET_KEY=1n53cur3K3y
+DJANGO_LOG_LEVEL="DEBUG"
```

Comment:
Is this one meant to be different to the other .envs?

Reply:
Yup - I thought that this is the ones devs start with, and they are likely to want to see debug messages.